.github/workflows/security_checks.yml

```diff
@@ -34,6 +40,8 @@ jobs:
         # file path to analyze
         GDN_BANDIT_TARGET: 'azext_edge'
         GDN_BANDIT_RECURSIVE: true
+        # checkov disable - build workflow has workflow_dispatch inputs
+        GDN_CHECKOV_SKIPCHECK: CKV_GHA_7
```

Comment:
We confirmed there is no security impact here? With our existing inputs for workflow_dispatch.

Reply:
I verified that there is no current security impact but on second thought, I'm on the fence about skipping this in our checks.

Currently, our `release_workflow` inputs are all booleans, that control different job stages:
<img width="436" alt="image" src="https://github.com/user-attachments/assets/f7c7f9d4-55f4-45d6-b718-d987b08d94a3">
None of these inputs are passed to the actual build workflow (`release_build.yml`) so we're good there.

Same goes for container build (just changes which tags are applied) and int tests (no artifacts produced).

However I still think we may want to keep these checks in and just dismiss / triage the alerts as they come, in case something gets added in the future that we might miss without this check running.